tests/test_swagger.py

```diff
@@ -164,3 +203,21 @@ def test_swagger_info(test_client, loop, swagger_info):
     assert '/example1' in result['paths']
     assert '/example2' in result['paths']
     assert 'API Title' in result['info']['title']
+
+@asyncio.coroutine
+def test_class_view(test_client, loop):
```

Comment:
Hey, your test does not actually check if the swagger.json is actually generated with the information provided in the doc strings